docs/guides/gunw_product_guide.md

```diff
@@ -24,39 +26,54 @@ You can download existing ARIA-S1-GUNW products from the Alaska Satellite Facili
+- [Ascending ARIA Frame IDs](https://d3g9emy65n853h.cloudfront.net/ARIA_S1_GUNW/ascending.geojson){target=_blank} 
+- [Descending ARIA Frame IDs](https://d3g9emy65n853h.cloudfront.net/ARIA_S1_GUNW/descending.geojson){target=_blank}
 
-To ensure that ARIA-S1-GUNW products are always created using the standard footprints, the ARIA Frame ID along with the reference and secondary granules that intersect this footprint for a given date need to be provided in order to create a new ARIA-S1-GUNW product (see figure below).
+### Search for Sentinel-1 SLCs for an ARIA Frame ID
 
-![Frame vs granule geographic footprint](../images/frame_granule_overlap.png "Example of a frame that spans three granules.")
+1. Search for Sentinel-1 SLC IW products in your area of interest in [Vertex](https://search.asf.alaska.edu/#/){target=_blank} using a Geographic Search and setting the Area of Interest to the desired ARIA Frame ID, as delineated in the [ARIA Frame ID maps](#aria-frame-id-maps). You may want to apply a search filter for the orbit direction that matches the ARIA Frame ID extent that you are using. 
 
-It can be tricky to find the appropriate granules for a given ARIA Frame ID, and in the future, we plan to create utilities to simplify this process. For the meantime, a geojson detailing the *ascending* ARIA Frame IDs can be downloaded [here](https://d3g9emy65n853h.cloudfront.net/ARIA_S1_GUNW/ascending.geojson){target=_blank} and a geojson detailing the *descending* ARIA Frame IDs can be downloaded [here](https://d3g9emy65n853h.cloudfront.net/ARIA_S1_GUNW/descending.geojson){target=_blank}.
+2. For each footprint that intersects the ARIA Frame ID, use the [SBAS](https://docs.asf.alaska.edu/vertex/sbas/){target=_blank} or [Baseline](https://docs.asf.alaska.edu/vertex/baseline/){target=_blank} tool in Vertex to find other acquisitions to pair with the reference acquisition.
 
-{% endblock %}
+    - You will need to repeat the process of finding pairs for each footprint that intersects the ARIA Frame ID extent
+
+3. Create a list of the primary and secondary Sentinel-1 IW SLCs that intersect with the ARIA Frame ID extent. 
+    
+    Example:
+    ```
+    ##TODO: Example list of input granules [primary1, primary2, primary3] [second1, second2, second3]
+    ```
+
+### Submit On-Demand ARIA-S1-GUNW Jobs
+
+!!! warning "On Demand support not currently available in Vertex for ARIA-S1-GUNW products"
+
+    On-demand ARIA S1 GUNW products cannot currently be submitted directly from Vertex, but we plan to make this feature available in the second half of 2025. Vertex is still very useful for selecting Sentinel-1 SLC pairs to submit for processing, but once you identify scene pairs, you will need to submit them using the [HyP3 Python SDK](../using/sdk.md){target=_blank} or [HyP3 API](../using/api.md){target=_blank}.
+
+  ##TODO: outline use of SDK and API for submitting jobs. 
 
-{% block packaging %}
 ## Product Packaging
 
 ### Naming convention
```

Comment:
Is the naming convention the same for the archived and on-demand GUNWs? I recall some discussion about having those generated by us distinguished from the archived data in some way, but I don't see anything in the naming scheme section that indicates a processing platform. Is that reflected in the product version tag at all? If so, that would be worth calling out.

Reply:
The naming convention is the same for GUNWs generated on-demand at the DAAC and GUNWs generated by the ARIA project. All products added to the DAAC archive from either system are created using the [standard set of processing parameters](https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/tree/dev?tab=readme-ov-file#what-makes-an-aria-s1-gunw-product-standard).